shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -264,6 +264,7 @@ - (void)installSplashScreenViewIfNecessary {
 }
 
 - (void)callViewRenderedCallback {
+  [self firstFrameDidRender];
```

Comment:
In here in the implementation would be an interface:
`@property(nonatomic, readwrite) BOOL firstFrameRendered;`
and here would be:
self.firstFrameRendered = YES;

Reply:
One thing we need to be able to do is set this and unset it each time the viewcontroller appears/disappears.  Will that still work with a property?

Reply:
Assuming the property is always set in the view controller implementation, then yes.

Reply:
In that case a better name might be `isRenderingFrames`.  If that works for @albertwant0116 I'll make that change.

Reply:
Just went ahead and made them - I think this makes themost sense.

Reply:
`isRenderingFrames` sounds to me that it changes the value for each frame lifecycle. But it actually indicates the first frame (after view controller being loaded) is rendering?

Reply:
It will be set to YES after at least one frame has rendered after showing the view controller, and set to now upon dismissing the view controller.

Right now, it's not guaranteed to only go YES / NO / YES / NO but it sounds like Jenn's suggestion could fix that.